relay-dynamic-config/src/global.rs

```diff
@@ -23,6 +25,9 @@ pub struct GlobalConfig {
     /// Quotas that apply to all projects.
     #[serde(skip_serializing_if = "Vec::is_empty")]
     pub quotas: Vec<Quota>,
+    /// Configuration for global inbound filters.
+    #[serde(skip_serializing_if = "skip_default_error_boundary")]
+    pub filters: ErrorBoundary<GenericFiltersConfig>,
```

Comment:
We can rename the variable and set a serde `rename`, but this seemed too much to me. What do you think?

The reason to keep `filters` in the protocol is because "inbound" is a `sentry` product term and Relay doesn't have to know about it. If new non-inbound filters are introduced, Relay may run them at this point.

Reply:
For my taste it's a bit too generic on the protocol level, but if there is a good reason to do it 👍

Reply:
In project configs, the field is `filterSettings`. I'm ok with updating the protocol to `inboundFilters`, but that also generates a consistency mismatch. What do you think?

Reply:
I'd prefer `inboundFilter` but maybe worth just bringing up with the team tomorrow.

Reply:
For consistency with the rest of Relay, we're sticking with `filters`.